pxr/usdImaging/usdImagingGL/CMakeLists.txt

```diff
@@ -142,3 +152,4411 @@ pxr_build_test(testUsdImagingGLSurfaceShader
         unitTestGLDrawing.cpp
         testenv/testUsdImagingGLSurfaceShader.cpp
 )
+
+if (${PXR_HEADLESS_TEST_MODE})
```

Comment:
Same comment as in testusdview: we should skip all of these tests on macOS for now. Maybe something like this?
```
if (MACOS)
    message(STATUS "Skipping ${PXR_PACKAGE} tests because they are currently unsupported on macOS")
    return()
endif()
```